src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.ts

```diff
@@ -188,7 +188,6 @@ export class TokenDetails {
       .ensure<IToken, string>(data => data.address)
       .required()
       .satisfiesRule(Validation.isETHAddress)
-      .then()
```

Comment:
```
Validate subsequent rules after previously declared rules have been validated successfully. 
Use to postpone validation of costly rules until less expensive rules pass validation.
```
Caused the following ERC20 rule not to trigger, thus the back.

---

The type of one-line fixes, that cause you hours of debugging :)